speaker/bgp_controller.go

```diff
@@ -295,9 +296,11 @@ func (c *bgpController) SetNode(l log.Logger, node *v1.Node) error {
 }
 
 var newBGP = func(logger log.Logger, addr string, srcAddr net.IP, myASN uint32, routerID net.IP, asn uint32, hold time.Duration, password string, myNode string, bgpType string) (session, error) {
+	level.Info(logger).Log("op", "startup", "msg", "Starting BGP session", "type", bgpType)
 	if bgpType == "native" {
-		level.Info(logger).Log("op", "startup", "msg", "Starting BGP session", "type", bgpType)
-		return native.New(logger, addr, srcAddr, myASN, routerID, asn, hold, password, myNode)
+		return bgpnative.New(logger, addr, srcAddr, myASN, routerID, asn, hold, password, myNode)
```

Comment:
@msherif1234 Had a comment about consistency of the naming convention for ASNs. I will address that in a later commit.